src/server/protocol.ts

```diff
@@ -3643,6 +3643,13 @@ export interface UserPreferences {
      * Default: `false`
      */
     readonly organizeImportsCaseFirst?: "upper" | "lower" | false;
+    /**
+     * Indicates where named type-only imports should sort. "inline" sorts imports without regard to if the import is

```

Comment:
When you add the VS Code option for this, make sure to say “type-only named imports” or “type-only import specifiers” since “type-only imports” could also refer to the relative order of these:

```ts
import type a from "./a";
import b from "./b";
```

which IIUC is unaffected by this preference.